internal/cmd/beta/security-group/create/create.go

```diff
@@ -0,0 +1,180 @@
+package create
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+)
+
+const (
+	nameFlag        = "name"
+	descriptionFlag = "description"
+	statefulFlag    = "stateful"
+	labelsFlag      = "labels"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	Labels      *map[string]any
+	Description string
+	Name        string
+	Stateful    bool
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates security groups",
+		Long:  "Creates security groups.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(`Create a named group`, `$ stackit beta security-group create --name my-new-group`),
+			examples.NewExample(`Create a named group with labels`, `$ stackit beta security-group create --name my-new-group --labels label1=value1,label2=value2`),
+		),
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create the security group %q?", model.Name)
+				err = p.PromptForConfirmation(prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			request := buildRequest(ctx, model, apiClient)
+
+			operationState := "Created"
+			p.Info("%s security group %q for %q\n", operationState, model.Name, model.ProjectId)
```

Comment:
This can be simplified to:

```suggestion
			p.Info("Created security group %q for %q\n", model.Name, model.ProjectId)
```

Reply:
Same for other commands

Reply:
Applied suggestion